cmd/gateway/commands.go

```diff
@@ -198,6 +212,27 @@ func createStaticModeCommand() *cobra.Command {
 		"Update the status of the GatewayClass resource.",
 	)
 
+	cmd.Flags().BoolVar(
+		&enableMetrics,
+		"enable-metrics",
+		false,
+		"Enable exposing metrics in the Prometheus format.",
+	)
+
+	cmd.Flags().IntVar(
+		&metricsListenPort,
+		"metrics-listen-port",
+		9113,
+		"Set the port where the metrics are exposed. [1023 - 65535]",
+	)
+
+	cmd.Flags().BoolVar(
+		&metricsSecure,
+		"metrics-secure-serving",
+		false,
+		"Enables serving metrics via https. By default metrics are served via http.",
```

Comment:
Should we be secure by default?

Reply:
I don't think so? The default scrape config from Prometheus is http, and because we're using a self-signed certificate, it requires extra config on the Prometheus side to enable the secure serving.